.github/workflows/tests.yml

```
@@ -8,6 +8,16 @@ on:
       - trying
       - staging
       - main
+on:
```

actionlint (1.7.7)

key "on" is duplicated in workflow. previously defined at line:3,col:1

(syntax-check)